server/utils/lists/content-index.js

```diff
@@ -0,0 +1,137 @@
+
+import { stripHtml } from "string-strip-html"; 
+import { camelCase } from 'change-case/keys';
```

Comment:
wahy not use lodash one instead so we all use the same one the same bahaviour

Reply:
Actually we use change-case more,  24 instances used of cc.camelCase vs 11 of _.camelCase